web/src/main/java/no/nav/modiapersonoversikt/rest/persondata/PersondataService.kt

```diff
@@ -58,11 +58,17 @@ class PersondataServiceImpl(
         val tilganger = PersondataResult
             .runCatching(InformasjonElement.VEILEDER_ROLLER) { hentTilganger() }
             .getOrElse(PersondataService.Tilganger(kode6 = false, kode7 = false))
+        val kontaktinformasjonTredjepartsperson = PersondataResult.runCatching(InformasjonElement.DKIF) {
```

Comment:
Tror jeg ville gått for noe alà: 
```kotlin
```kotlin
private fun hentKontaktInformasjonForTredjepartsPersoner(person: HentPersondata.Person): PersondataResult<Map<String, Dkif.DigitalKontaktinformasjonTredjepartsperson>> {
        return PersondataResult.runCatching(InformasjonElement.DKIF_TREDJEPART) {
            person
            .findKontaktinformasjonTredjepartspersoner()
            .associateWith { dkif.hentDigitalKontaktinformasjon(it) }
            .mapValues { entry ->
                Dkif.DigitalKontaktinformasjonTredjepartsperson(
                    mobiltelefonnummer = entry.value.mobiltelefonnummer?.value,
                    reservasjon = entry.value.reservasjon
                )
            }
        }
    }
```
```